pyEpiabm/pyEpiabm/property/place_foi.py

```diff
@@ -55,16 +55,14 @@ def place_inf(place, infector, time: float):
         return place_inf
 
     @staticmethod
-    def place_susc(place, infector, infectee,
+    def place_susc(place, infectee,
                    time: float):
```

Comment:
```suggestion
    def place_susc(place, infectee, time: float):
```

No need to split over lines now I think, but tiny change!